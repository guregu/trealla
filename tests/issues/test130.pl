:-initialization(main).

main :-
	call_cleanup(between(1,5,_), throw(error)), (between(1,5,_) -> !).
