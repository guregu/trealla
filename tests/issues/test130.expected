   Warning: Initialization goal exception: error.
